src/Synapse/Mapper/FinderTrait.php

```diff
@@ -4,7 +4,7 @@
 
 use InvalidArgumentException;
 use LogicException;
-use Synapse\Mapper\PaginationData;
```

Comment:
Looks like `PaginationData` was `use`d here as a FQCN reference because it appears in a `@return` statement.

In other words, the reason it was there in the first place is the same reason that you added the `use` statement for `AbstractEntity` right below.

I don't care one way or the other, but we should be consistent.